R/radsl.R

```diff
@@ -16,6 +16,9 @@
 #' @inheritParams argument_convention
 #' @param N (`numeric`)\cr Number of patients.
 #' @param study_duration (`numeric`)\cr Duration of study in years.
+#' @param race_list character vector of RACE
+#' @param race_prob numeric vector of probabilities of being assigned to a RACE
+#' @param female_prob (`proportion`)\cr Probability of a subject being female, male is calculated by `1-female_prob`.
```

Comment:
Please try to keep the convention:
- start with type of the data
- Full sentence starting with a capital and ends with a dot.